packages/@lwc/module-resolver/src/__tests__/fixtures/unrelated/lwc.config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@diervo I think the `unrelated` directory can be deleted. I had a test that used this fixture that I removed and forgot to remove this directory.